app/src/main/java/org/liberty/android/fantastischmemo/dao/CardDaoImpl.java

```diff
@@ -858,5 +869,67 @@ private void maintainOrdinal() throws SQLException {
             throw new RuntimeException(e);
         }
     }
-}
+
+    public void shuffleCards(){
+        List<Card> cards = queryForAll();
+
+        Card[] cs = cards.toArray(new Card[cards.size()]);
```

Comment:
we should probably have our variable names to be more meaningful. So change 'cs'